docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
-    image: node:current
+    image: node:10.16.3
```

Comment:
Hm, I just noticed that we specify both `build` and `image` here, and `build` takes precedence. We should actually drop this, and update https://github.com/mozilla-frontend-infra/balrog-ui/blob/master/Dockerfile#L1.

Can we pin to `node:10` instead, too? That way we'll pick up minor updates.